Automattic/newspack-plugin
# feat(ads): integrate Broadstreet into the providers wizard

### All Submissions:






### Changes proposed in this Pull Request:

Removes WordAds and AdSense integration from Setup and Ads wizards.

![image](https://user-images.githubusercontent.com/820752/150855758-20d5c4a7-3756-413a-9d6a-f450fe9df0d4.png)

![image](https://user-images.githubusercontent.com/820752/150855857-45e61f96-f3f9-44cb-a3f4-0aaf39eed06e.png)

Closes https://github.com/Automattic/newspack-ads/issues/249.

### How to test the changes in this Pull Request:

1. Visit Advertising home wizard and WordAds and AdSense should no longer be available as options
2. Start a new site, complete the onboarding setup and confirm the Google AdSense was not an available "extra feature"

### Other information:
